test/integration/good/function-signatures/distributions/univariate/continuous/chi_square/chi_square_cdf.stan

```diff
@@ -8,31 +8,31 @@ data {
 }
 transformed data {
   real transformed_data_real;
-  transformed_data_real = chi_square_cdf(d_int, d_int);
-  transformed_data_real = chi_square_cdf(d_int, d_real);
-  transformed_data_real = chi_square_cdf(d_int, d_real_array);
-  transformed_data_real = chi_square_cdf(d_int, d_vector);
-  transformed_data_real = chi_square_cdf(d_int, d_row_vector);
-  transformed_data_real = chi_square_cdf(d_real, d_int);
-  transformed_data_real = chi_square_cdf(d_real, d_real);
-  transformed_data_real = chi_square_cdf(d_real, d_real_array);
-  transformed_data_real = chi_square_cdf(d_real, d_vector);
-  transformed_data_real = chi_square_cdf(d_real, d_row_vector);
-  transformed_data_real = chi_square_cdf(d_real_array, d_int);
-  transformed_data_real = chi_square_cdf(d_real_array, d_real);
-  transformed_data_real = chi_square_cdf(d_real_array, d_real_array);
-  transformed_data_real = chi_square_cdf(d_real_array, d_vector);
-  transformed_data_real = chi_square_cdf(d_real_array, d_row_vector);
-  transformed_data_real = chi_square_cdf(d_vector, d_int);
-  transformed_data_real = chi_square_cdf(d_vector, d_real);
-  transformed_data_real = chi_square_cdf(d_vector, d_real_array);
-  transformed_data_real = chi_square_cdf(d_vector, d_vector);
-  transformed_data_real = chi_square_cdf(d_vector, d_row_vector);
-  transformed_data_real = chi_square_cdf(d_row_vector, d_int);
-  transformed_data_real = chi_square_cdf(d_row_vector, d_real);
-  transformed_data_real = chi_square_cdf(d_row_vector, d_real_array);
-  transformed_data_real = chi_square_cdf(d_row_vector, d_vector);
-  transformed_data_real = chi_square_cdf(d_row_vector, d_row_vector);
+  transformed_data_real = chi_square_cdf(d_int |  d_int);
```

Comment:
Two spaces after `|` just bugs me. (And how did you change these all without a working canonicalizer?)

Reply:
Rok taught me about vscode's regex find and replace and we used that. It's amazing.

Reply:
Canonicalizer would have been much simpler :) Well...

Reply:
Ran 

```
dune build
for i in $(find test/integration/good/function-signatures/distributions/ -name "*_cdf*.stan")
 do
 ./_build/default/src/stanc/stanc.exe --print-canonical $i 1>temp.txt 2>/dev/null
 mv temp.txt $i
 done
```
so should be nicer now.